unwind/src/lib.rs

```diff
@@ -9,7 +9,7 @@ extern crate fallible_iterator;
 use gimli::{UnwindSection, UnwindTable, UnwindTableRow, EhFrame, BaseAddresses, UninitializedUnwindContext, Pointer, Reader, EndianSlice, NativeEndian, CfaRule, RegisterRule, EhFrameHdr, ParsedEhFrameHdr, X86_64};
 use fallible_iterator::FallibleIterator;
 
-mod registers;
+pub mod registers;
```

Comment:
Not sure the extra namespace here provides any value, maybe just expose `Registers` instead?

Also I don't know why `mod glue` is public.

Reply:
> Not sure the extra namespace here provides any value, maybe just expose Registers instead?

Done

Reply:
> Also I don't know why `mod glue` is public.

@philipc `mod glue` is public because rustc removes even an extern fn if it is neither referenced inside the crate nor callable from outside Rust code. Since the unwind recorder is invoked through external asm rustc would assume the function is dead and remove it.